apps/frontend/src/components/base/Card.svelte

```diff
@@ -1,3 +1,9 @@
-<div class="p-4 bg-gray-50 rounded-md shadow-md">
+<script lang="ts">
+    export let classExtra: string | undefined;
```

Comment:
Idk if its worth, but this can be like classesExtra as string[]. You can then call join with a " ". Or just classesExtra as a string which would be space seperated

Reply:
could probably do a `string | string[]` where in the latter case it does the join.